src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -256,9 +256,14 @@ public static function fromDsn(string $dsn, array $redisOptions = [], $redis = n
                 $connectionCredentials['host'] = 'tls://'.$connectionCredentials['host'];
             }
         } else {
+            $pass = '' !== ($parsedUrl['pass'] ?? '') ? $parsedUrl['pass'] : null;
+            $user = '' !== ($parsedUrl['user'] ?? '') ? $parsedUrl['user'] : null;
```

Comment:
looks like this is duplicated from the previous "if". Can we move all this outside the "if/else"?

Reply:
Done.